<?xml version="1.0"?>
<launch>
  <!-- 參數 -->
  <arg name="plane_a" default="2.0" />
  <arg name="plane_b" default="1.5" />
  <arg name="plane_x" default="0.0" />
  <arg name="plane_y" default="0.0" />
  <arg name="plane_z" default="0.0" />

  <!-- 平面顯示節點 -->
  <node name="plane_visualizer" pkg="viz_pkg" type="plane_visualizer.py" output="screen">
    <param name="plane_a" value="$(arg plane_a)" />
    <param name="plane_b" value="$(arg plane_b)" />
    <param name="plane_x" value="$(arg plane_x)" />
    <param name="plane_y" value="$(arg plane_y)" />
    <param name="plane_z" value="$(arg plane_z)" />
  </node>

  <!-- rviz -->
  <node name="rviz" pkg="rviz" type="rviz"
        args="-d $(find viz_pkg)/rviz/plane_visualization.rviz" output="screen" />

</launch>
